nixos/modules/programs/fish.nix

```diff
@@ -169,18 +178,75 @@ in
       end
     '';
 
+    programs.fish.interactiveShellInit = ''
```

Comment:
Will this be overwritten if someone sets `programs.fish.interactiveShellInit` in their `configuration.nix`?

Reply:
The `programs.fish.interactiveShellInit` option has type `lines`, so no, they will be concatenated together.